libportal/notification.c

```diff
@@ -306,3 +306,44 @@ xdp_portal_remove_notification (XdpPortal  *portal,
                           NULL,
                           NULL);
 }
+
+/**
+ * xdp_portal_get_supported_options:
+ * @portal: a [class@Portal]
+ *
+ * Returns: a vardict of supported options for properties that have options.
+ *
+ * Since: 0.7.2
+ */
+GVariant *
+xdp_portal_get_supported_options (XdpPortal *portal)
```

Comment:
Hm, I'm not particularly fond of just exposing the raw GVariant... I think either libportal provides some "added value" by exposing a nice API around it (XdpNotificationOptions?), or we let the app check that using their favorite tools. This is a simple D-Bus call anyway.

(Also, `xdp_get_supported_notification_options` is probably a better name?)

Reply:
> This is a simple D-Bus call anyway.

Isn't that the case for the entire portal? Anyhow I'm fine with adding `XdpNotificationOptions`

> (Also, xdp_get_supported_notification_options is probably a better name?)

yes, definitely.

Reply:
As discussed on matrix let's not create to much abstraction over the interface.